### PR TITLE
Add new jupyterlab image to dependabot configuration

### PR DESCRIPTION
**Pull request by jonatanschroeder**
The new jupyterlab image was not added to dependabot configuration.

### CODE REVIEW

**Review by eliotwrobson — APPROVED** [review]

**Comment by jonatanschroeder** [comment]
Probably good to wait until the Oct 30 or Nov 1 to merge this, since dependabot tends to re-run when the configuration file changes...

**Comment by nwalters512** [comment]
I've manually bumped a lot of dependencies, so I'm going to merge this now. Hopefully we don't get too much additional noise!

**Pull request merged**
